### DISCUSSION TITLE
TestRunner PowerShell repo?

### DISCUSSION THREAD

**Discussion by slaven** [microsoft/ALAppExtensions#22526] [Q&A]
Hello,

Where exactly is official repo for TestRunner PowerShell scripts?

From what I can see, PowerShell scripts which are included in official BC installation: "Dynamics.365.BC.xxxxx.W1.DVD.zip" under "Applications\testframework\TestRunner" are not included here in this repo.

I'd like to report an issue regarding "RunBCPTTests.ps1" or "ALTestRunner.psm1" scripts and I don't see where or how can I do that?

Thank you

**Comment by JesperSchulz** [comment]
Those are not open sourced. @nikolakukrika, would you be willing to accept an issue report here on GitHub?

**Reply by slaven** [reply]
That's exactly why I asked this question.
Scripts are not open sourced but the source itself is visible and debuggable because we're talking about PowerShell scripts which are basically plain text.
I don't see any reason why nobody can reference, contribute or create an issue regarding any of TestRunner scripts.
For example, maybe I just want to see diffs between versions.  Instead of checking diff between commits on github, I need to download 1TB of artifacts. Why?
